src/Interpreters/ExpressionAnalyzer.cpp

```diff
@@ -155,12 +170,7 @@ bool checkPositionalArguments(ASTPtr & argument, const ASTSelectQuery * select_q
                 }
             }
             else if (pos > columns.size() || !pos)
-            {
-                throw Exception(ErrorCodes::ILLEGAL_TYPE_OF_ARGUMENT,
```

Comment:
As I understand now we will ignore incorrect positional arguments?

Reply:
If out of bound.

Reply:
ok